docs/topics/auth/customizing.txt

```diff
@@ -745,12 +745,13 @@ providing two additional methods:
         model uses ``email`` as the username field, and has ``date_of_birth``
         as a required field, then ``create_superuser`` should be defined as::
 
-            def create_superuser(self, email, date_of_birth, password):
+            def create_superuser(self, email, date_of_birth, password=None):
                 # create superuser here
                 ...
 
-        Unlike ``create_user()``, ``create_superuser()`` *must* require the
-        caller to provide a password.
+        Like ``create_user()``, ``create_superuser()`` does not require the
+        caller to provide a password. However, it's still a good idea to
+        provide one, as superusers have unrestricted access to all site data.
```

Comment:
Is this right? If you don't provide a password the user just can't log in no? 

(The `...unrestricted access...` bit seems to imply that they could log in without a password if you don't set one. That **would** be a worry.) 

Could we just drop this paragraph?

Reply:
No. It is not. I just traced the code back to where the password is set. Aside from the admin form not accepting blank passwords, if a password is not provided, a random (unusable) one is set.

The second sentence is false, but the first sentence now just seems out of place. I'll drop the whole paragraph.